### DISCUSSION TITLE
Copy & paste

### DISCUSSION THREAD

**Discussion by InsaneCake** [teableio/teable#1085] [Q&A]
Is it possible to make copy&paste functionality working with http for LAN selfhosted cases?
For exsample as optional ENV variable or switch in instance admin panel.
If not that is best way to make local https connection?

**Comment by tea-artist** [comment]
We have adjusted the implementation of the copy and paste feature. Now, even if https is not enabled, the copy should work, However, if the copied area exceeds the currently loaded data range (Usually 300rows), https still needs to be enabled to take effect.

**Comment by InsaneCake** [comment]
In 1.5.2 it works for the records, but copy button on share link still doesn't work. Just silently do nothing.
![image](https://github.com/user-attachments/assets/1c42455d-d186-4afa-a45c-d0daf244e5d7)

**Reply by tea-artist** [reply]
cc @boris-w

**Comment by InsaneCake** [comment]
Works in 1.6.0 release
